src/Microsoft.AspNetCore.SignalR.Core/Internal/AsyncEnumeratorAdapters.cs

```diff
@@ -62,8 +62,8 @@ public void OnNext(T value)
                     // Wait for a spot
                     if (!_output.WaitToWriteAsync(_cancellationToken).Result)
```

Comment:
Off topic: `.Result`? I'm guessing because `OnNext` is not async. Should there be a comment explaining why and consequences?

Reply:
Hm, that's what it seems like to me. Doesn't seem necessary but I'll add it if people want it

Reply:
Yeah. Also because we use unbounded channels, this will never go async.

Reply:
If the token is canceled, this will throw `OperationCanceledException`, we wouldn't want that leaking out either, right? Should we just wrap the thing in a `try`/`catch`?

Reply:
I spoke to @mikaelm12 , he's fixing it.

Reply:
Though depending on #1864 , it may start mattering again later. We can deal with that when we resolve that issue though.